reactos/drivers/usb/usbhub_new/usbhub.c

```diff
@@ -1679,15 +1678,9 @@ USBH_ChangeIndicationProcessChange(IN PDEVICE_OBJECT DeviceObject,
     }
     else
     {
-        WorkItem = HubExtension->WorkItemToQueue;
-
-        if (!HubExtension->WorkItemToQueue)
-        {
-            DPRINT1("USBH_ChangeIndicationProcessChange: WorkItem == NULL \n");
-            KeBugCheckEx(BUGCODE_USB_DRIVER, 0xC1, 0, 0, 0);
-            return STATUS_MORE_PROCESSING_REQUIRED;
-        }
+        ASSERT("HubExtension->WorkItemToQueue != NULL");
```

Comment:
ASSERT takes a condition as its argument, you'd want `ASSERT(HubExtension->WorkItemToQueue != NULL)`
A string will always evaluate to true, so the assert will never fire.